src/coinbase/staking_reward.ts

```diff
@@ -109,12 +109,39 @@ export class StakingReward {
     return this.model.address_id;
   }
 
+  /**
+   * Returns the USD value of the StakingReward.
+   *
+   * @returns The USD value.
+   */
+  public usdValue(): Amount {
+    return new Decimal(this.model.usd_value.amount).div(new Decimal("100"));
+  }
+
+  /**
+   * Returns the conversion price of the StakingReward in USD.
+   *
+   * @returns The conversion price.
+   */
+  public conversionPrice(): Amount {
+    return new Decimal(this.model.usd_value.conversion_price);
+  }
+
+  /**
+   * Returns the time of calculating the conversion price.
+   *
+   * @returns The conversion time.
+   */
+  public conversionTime(): Date {
+    return new Date(this.model.usd_value.conversion_time);
+  }
+
   /**
    * Print the Staking Reward as a string.
    *
    * @returns The string representation of the Staking Reward.
    */
   public toString(): string {
-    return `StakingReward { date: '${this.date().toISOString()}' address: '${this.addressId()}' amount: '${this.amount().toString()}' }`;
+    return `StakingReward { date: '${this.date().toISOString()}' address: '${this.addressId()}' amount: '${this.amount().toString()}' usd_value: '${this.usdValue().toString()}' conversion_price: '${this.conversionPrice().toString()}' conversion_time: '${this.conversionTime().toISOString()}' }`;
```

Comment:
Does this function call still print out logically in the case where a user asks for native units?

Reply:
I'll add a test case for this

But it'll always print out conversion data regardless of the user provided format.

Reply:
Would you like it to behave differently if user asks for native units (not provide conversion data?)

Reply:
This is how it looks when the user supplies `native`

```
> let rewards = await StakingReward.list( Coinbase.networks.EthereumMainnet , Coinbase.assets.Eth,["0x60c7e246344ae3856cf9abe3a2e258d495fc39e0"],"2024-05-01T00:00:00Z","2024-05-21T00:00:00Z", "native");
rewards.forEach(reward => {console.log(reward.toString())});undefined
> rewards.forEach(reward => {console.log(reward.toString())});
StakingReward { date: '2024-05-01T00:00:00.000Z' address: '0x60c7e246344ae3856cf9abe3a2e258d495fc39e0' amount: '0.001212525541415161' usd_value: '3.61' conversion_price: '2971.419922' conversion_time: '2024-05-02T00:09:00.000Z' }
StakingReward { date: '2024-05-02T00:00:00.000Z' address: '0x60c7e246344ae3856cf9abe3a2e258d495fc39e0' amount: '0.000679464305235174' usd_value: '2.03' conversion_price: '2984.810059' conversion_time: '2024-05-03T00:09:00.000Z' }
StakingReward { date: '2024-05-03T00:00:00.000Z' address: '0x60c7e246344ae3856cf9abe3a2e258d495fc39e0' amount: '0.000726030823305604' usd_value: '2.26' conversion_price: '3103.570068' conversion_time: '2024-05-04T00:09:00.000Z' }
StakingReward { date: '2024-05-04T00:00:00.000Z' address: '0x60c7e246344ae3856cf9abe3a2e258d495fc39e0' amount: '0.000681550080574926' usd_value: '2.13' conversion_price: '3116.860107' conversion_time: '2024-05-05T00:09:00.000Z' }
StakingReward { date: '2024-05-05T00:00:00.000Z' address: '0x60c7e246344ae3856cf9abe3a2e258d495fc39e0' amount: '0.000609926334576295' usd_value: '1.92' conversion_price: '3144.01001' conversion_time: '2024-05-06T00:09:00.000Z' }
StakingReward { date: '2024-05-06T00:00:00.000Z' address: '0x60c7e246344ae3856cf9abe3a2e258d495fc39e0' amount: '0.000618244877949755' usd_value: '1.9' conversion_price: '3070.639893' conversion_time: '2024-05-07T00:09:00.000Z' }
StakingReward { date: '2024-05-07T00:00:00.000Z' address: '0x60c7e246344ae3856cf9abe3a2e258d495fc39e0' amount: '0.000604674331782671' usd_value: '1.83' conversion_price: '3016.580078' conversion_time: '2024-05-08T00:09:00.000Z' }
StakingReward { date: '2024-05-08T00:00:00.000Z' address: '0x60c7e246344ae3856cf9abe3a2e258d495fc39e0' amount: '0.000631405122359757' usd_value: '1.88' conversion_price: '2976.870117' conversion_time: '2024-05-09T00:09:00.000Z' }
StakingReward { date: '2024-05-09T00:00:00.000Z' address: '0x60c7e246344ae3856cf9abe3a2e258d495fc39e0' amount: '0.00070716574809992' usd_value: '2.15' conversion_price: '3033.02002' conversion_time: '2024-05-10T00:09:00.000Z' }
StakingReward { date: '2024-05-10T00:00:00.000Z' address: '0x60c7e246344ae3856cf9abe3a2e258d495fc39e0' amount: '0.000552724206304093' usd_value: '1.61' conversion_price: '2908.919922' conversion_time: '2024-05-11T00:09:00.000Z' }
```

Reply:
I think we should avoid something like this:

```
date: <date> address: <address> count: <count> usd_value:  conversion_price:  conversion_time:
```

getting printed, where the USD information has empty values. As long as we're not printing our empty values, then I'm good to approve 👍.

Reply:
Yeah it'll never be empty, RR always provides USD data, so does CDP service